.textlintrc.json

```
@@ -83,10 +83,12 @@
         "YouTube",
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
         ["3rd[- ]party", "third-party"],
-        ["bug[- ]fix(es)?", "bugfix$1"],
         ["cpp", "C++"],
-        ["dotnet", ".NET"],
+        ["[Dd]ot[Nn]et", ".NET"],
+        ["EBPF", ".eBPF"],
+        ["eb[Pp][Ff]", ".eBPF"],
```

I suspect that the `.` was a copy-paste error. Also, maybe we can combine the two rules:

```suggestion
        ["[eE][bB][Pp][Ff]", "eBPF"],
```

---

I've adapted the suggestion above based on @svrnm's comment about the regex being case insensitive.